include/mbedtls/x509_crt.h

```diff
@@ -348,6 +459,30 @@ int mbedtls_x509_crt_parse_file( mbedtls_x509_crt *chain, const char *path );
 int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path );
 #endif /* MBEDTLS_FS_IO */
 
+/**
+ * \brief          Parses a subject alternative name raw item
+ *                 to an identified structure;
+ *
+ * \param san_raw  The buffer hlding the raw data of the subject alternative name.
```

Comment:
Minor: Typo `hlding` -> `holding`

Reply:
We should indicate that this must include the ASN.1 tag+length header. (Otherwise it wouldn't work anyway, but better say it nonetheless).

Reply:
but that's already part of `mbedtls_x509_buf `